src/commands/oracle/reports.ts

```diff
@@ -23,15 +23,16 @@ export default class Reports extends BaseCommand {
   static example = ['reports StableToken', 'reports', 'reports StableTokenEUR']
 
   async run() {
-    const res = this.parse(Reports)
-    const sortedOracles = await this.kit.contracts.getSortedOracles()
+    const kit = await this.getKit()
+    const res = await this.parse(Reports)
+    const sortedOracles = await kit.contracts.getSortedOracles()
 
-    const reports = await sortedOracles.getReports(res.args.token).catch((e) => failWith(e))
-    cli.table(
+    const reports: any = await sortedOracles.getReports(res.args.token).catch((e) => failWith(e))
+    CliUx.ux.table(
       reports,
       {
         address: {},
-        rate: { get: (r) => r.rate.toNumber() },
+        rate: { get: (r: any) => r.rate.toNumber() },
```

Comment:
nit: the `any` is unfortunate

Reply:
Yes I struggled for awhile with the CLI table API and was never able to get it to accept the typings. It will seem like it should absolutely work but still throws a compiler error so eventually I gave up.